flowexperimental/flowexp.hpp

```diff
@@ -25,8 +25,8 @@
  *
  * \brief The common settings for all ebos variants.
```

Comment:
should probably clean out references to ebos in this file (comments).

Reply:
Sure. Tried while missed this one.

Reply:
there are more.